docs/book/stacks-and-components/component-guide/orchestrators/sagemaker.md

```diff
@@ -119,6 +133,27 @@ side. From there you can view the pipelines that have been launched via ZenML:
 
 ![Sagemaker Studio Pipelines](../../../.gitbook/assets/sagemakerUI.png)
 
+### Debugging SageMaker Pipelines
+
+If your SageMaker pipeline failed before the first ZenML step was started, the
+ZenML run will not show up in the ZenML dashboard. In these cases, you need to 
+[use the SageMaker UI](#sagemaker-ui) to view the respective error message and
+logs. To do so, open the corresponding pipeline in the SageMaker UI, then open
+its only execution, then click on the failed step in the pipeline graph and go
+to the 'Output' tab:
```

Comment:
I personally find bullet points or numbered lists a better approach to describe series of steps but this paragraph is also great.
```suggestion
If your SageMaker pipeline encounters an error before the first ZenML step starts, the ZenML run will not appear in the ZenML dashboard. In such cases, use the [SageMaker UI](#sagemaker-ui) to review the error message and logs. Here's how:

Open the corresponding pipeline in the SageMaker UI.
Open its execution.
Click on the failed step in the pipeline graph.
Go to the 'Output' tab to access logs.
```

Reply:
Great suggestions 👍